lib/runtime/errors/invalid_transaction.go

```diff
@@ -0,0 +1,130 @@
+// Copyright 2022 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package errors
+
+import (
+	"github.com/ChainSafe/gossamer/pkg/scale"
+)
+
+// InvalidTransaction is child VDT of TransactionValidityError
+type InvalidTransaction scale.VaryingDataType
+
+// Index fulfils the VaryingDataTypeValue interface.  T
+func (InvalidTransaction) Index() uint {
+	return 0
+}
+
+// Call The call of the transaction is not expected
+type Call struct{}
+
+// Index Returns VDT index
+func (Call) Index() uint { return 0 }
+
+// Payment General error to do with the inability to pay some fees (e.g. account balance too low)
+type Payment struct{}
+
+// Index Returns VDT index
```

Comment:
supernit maybe batch rename `Index Returns VDT index` to `Index returns the VDT index`